### PR TITLE
Merge dotfiles

### LINKED ISSUE #20: Merge individual dotfiles into one
Dotfiles like `.eslintrc`, `.prettierrc` in individual sub-directories should probably be merged into one in root directory.

### PR DESCRIPTION
**Pull request by rzadp**
Merged / Moved to root dir:
- `.gitignore`
- `.editorconfig`
- `.nvmrc`
  - I chose 10.15 in order to follow `.nvmrc` in the contracts
- `.dockerignore`

Removed:
- `.nycrc` as nyc is not used

I think it closes #20

### CODE REVIEW

**Comment by rzadp** [comment]
> Should `oracle/.dockerignore` be used by `oracle/Dockerfile` when the oracle image is built for production?

Good call, I have duplicated the `.dockerignore` so now we have 2 `Dockerfile's` and 2 corresponding `.dockerignore's`

File: oracle/.dockerignore [hunk 1]
```diff
@@ -1,12 +1,6 @@
...
-.git*
-LICENSE
-README.md
-node_modules
+**/node_modules
+**/submodules
+**/Dockerfile
+**/.dockerignore
+**/docker-compose.yml
+.gitmodules
```
Comment (akolotov) [inline oracle/.dockerignore]: Do you think the following files and directories are needed to be included in the oracle docker image?
```
docs
test
README.md
*.png
```

Comment (rzadp) [reply oracle/.dockerignore]: Let's remove them in #66 , in which I will probably be able to use only one `.dockerfile`

Comment (rzadp) [reply oracle/.dockerignore]: I have updated the `.dockerignore` on both PRs.

File: .dockerignore [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
-**/**/.git
+**/submodules
+**/Dockerfile
+**/.dockerignore
+**/docker-compose.yml
 .gitmodules
+**/.git
+**/docs
+**/*.md
+**/*.png
```
Comment (akolotov) [inline .dockerignore]: Is it safe and will not exclude elements of UI in the tokenbridge UI?

Comment (rzadp) [reply .dockerignore]: Sorry I didn't think of it, it probably will.

**Pull request merged**